### PR TITLE
fix(core): Modify safeStringify to allow multiple null values

### LINKED ISSUE #4798: bug: Capacitor androidBridge.postMessage converting `null` to `"..."`
originally posted here but closed due to inactivity: https://github.com/ionic-team/capacitor/issues/4746

the method `safeStringify` in https://github.com/ionic-team/capacitor/blob/0429905efcf69097e7759b8d21d74459b8dd42c5/android/capacitor/src/main/assets/native-bridge.js#L175 does not account for `null` values when checking for circular references (`null` has the type `object` as well). therefore `[10,null,null,null]` is converted to `[10,null,'...','...']` (as posted in the original issue).

bug was introduced here: https://github.com/ionic-team/capacitor/commit/0429905efcf69097e7759b8d21d74459b8dd42c5#diff-8afda3e2bce7fb91a14a66d96d63a89f112b852cf39223e6f820a1575cbdafebR175

### PR DESCRIPTION
**Pull request by thomasvidas**
Fixes #4798

### CODE REVIEW

**Comment by thomasvidas** [comment]
Tested on this reproduction https://github.com/jepiqueau/capacitor-sqlite-test

Before fix
Screen Shot 2021-07-20 at 12 10 41 PM

After fix
Screen Shot 2021-07-20 at 12 10 03 PM

**Review by elylucas — APPROVED** [review]

**Comment by emirtb** [comment]
> Tested on this reproduction https://github.com/jepiqueau/capacitor-sqlite-test
> 
> Before fix
> Screen Shot 2021-07-20 at 12 10 41 PM
> 
> After fix
> Screen Shot 2021-07-20 at 12 10 03 PM

Is a null type or "null" string???

**Comment by thomasvidas** [comment]
It is a JSONObject with a value of `null`

File: core/native-bridge.ts [hunk 1]
```diff
@@ -200,7 +200,8 @@ const initBridge = (w: any): void => {
     const seen = new Set();
     return JSON.stringify(value, (_k, v) => {
       if (seen.has(v)) {
-        return '...';
+        if (v === null) return null;
```
Comment (herecoulbeyourname) [inline core/native-bridge.ts:203]: what about `undefined` values?

**Pull request merged**